cmd/start.go

```diff
@@ -155,25 +96,23 @@ func Start(c *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed while creating a storage disk handler: [%v]", err)
 	}
-	persistence := persistence.NewEncryptedPersistence(
+	encryptedPersistence := persistence.NewEncryptedPersistence(
 		handle,
 		config.Ethereum.Account.KeyFilePassword,
 	)
 
 	err = beacon.Initialize(
 		ctx,
-		operatorPublicKey,
-		chainProviderV1.ThresholdRelay(),
 		beaconChain,
 		netProvider,
-		persistence,
+		encryptedPersistence,
 	)
 	if err != nil {
 		return fmt.Errorf("error initializing beacon: [%v]", err)
 	}
 
-	initializeMetrics(ctx, config, netProvider, stakeMonitorV1, operatorPublicKey)
-	initializeDiagnostics(ctx, config, netProvider, chainProviderV1.Signing())
+	initializeMetrics(ctx, config, netProvider, blockCounter)
```

Comment:
Nice! This should let @tomaszslabon get rid of `StakeMonitor` completely in #3060.